src/data/hicdex.js

```diff
@@ -142,7 +142,7 @@ shareholder(where: {holder_id: {_eq: $address}, holder_type: {_eq: "core_partici
 }`
 
 export const getCollabTokensForAddressesByShare = `query GetCollabTokens($addresses: [String!]) {
-  split_contract(where: {shareholder: {holder_id: {_in: $addresses}}, contract: {tokens: {supply: {_gt: "0"}, _and: {is_signed: {_eq: true}}}}}) {
+  split_contract(where: {shareholder: {holder_id: {_in: [KT1DWnLiUkNtAQDErXxudFEH63JC6mqg3HEx]}}, contract: {tokens: {supply: {_gt: "0"}, _and: {is_signed: {_eq: true}}}}}) {
```

Comment:
wondering if is this correct...

Reply:
I don't think so. The $address var has to be used, or hasura will whine about it.
Also KT1DWnLiUkNtAQDErXxudFEH63JC6mqg3HEx should be "KT1DWnLiUkNtAQDErXxudFEH63JC6mqg3HEx".
Maybe this was hard coded for debugging purposes once?

Reply:
Nevermind, it "works" but it throws some warnings in the console :smile:

Reply:
I think I added it indeed for debugging reasons